services/src/test/java/org/keycloak/protocol/oidc/utils/RedirectUtilsTest.java

```diff
@@ -0,0 +1,117 @@
+package org.keycloak.protocol.oidc.utils;
```

Comment:
Look at OAuthRedirectUriTest which can be extended to cover these tests properly